README.md

```diff
@@ -56,6 +58,24 @@ pin-depends: [
 ...
 ```
 
+### Esy resolution
+
+The library can also be installed with [esy](esy.sh) using a resolution in your `package.json` file :
+
+```json
+...
+    "dependencies": {
+      ...
+      "@opam/preface":"*"
+    },
+    "resolutions": {
+        "@opam/preface":"xvw/preface#e02e1bb"
```

Comment:
Could you remove the commit avec put `<commit>` in order to make the README not so "obseletable". Since there is a link just below, I think it is not necessary to put the commit in the snipper.